lua/tardis/cl_menuoptions.lua

```diff
@@ -0,0 +1,201 @@
+hook.Add("PopulateToolMenu", "TARDIS2-PopulateToolMenu", function()
+    -- Options
+    local options={}
+    local sections={}
+    local subsections={}
+    for k,v in pairs(TARDIS:GetSettingsData()) do
+        if v.option then
+            table.insert(options,{v.id, v, (v.subsection or " ") .. v.name})
+            if v.section and not table.HasValue(sections,v.section) then
+                table.insert(sections, v.section)
+            end
+            if v.section and v.subsection then
+                subsections[v.section] = subsections[v.section] or {}
+                subsections[v.section][v.subsection] = true
+            end
+        end
+    end
+
+    table.SortByMember(options, 3, true)
+    table.SortByMember(sections, 1, true)
+
+    for k,section in ipairs(sections) do
+        local section_id = "TARDIS2_Options_" .. section
+        local section_text = " " .. section
```

Comment:
As in spawnmenu's custom settings, there are two types of spaces
The point of them is sorting the options
"Reset All Settings" needs to go down

Reply:
Do they not get added to the menu in the order they are added?

Reply:
Not sure, but they used to be sorted at some point
Also, sometimes the menus change order when you mess with the code
IMO, it's better to have control over what is where, since the list is generated automatically anyway

Reply:
Okay, it's not ideal but I get not something we have a lot of control over. Can always revisit this later if a better solution comes up